source/utils.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
         outputPath: path.join(__dirname, '..', 'static'),
         withCoverage: process.argv.indexOf('--with-acceptance') === -1 && process.argv.indexOf('--no-coverage') === -1,
         withLint: process.argv.indexOf('--no-lint') === -1,
+        withFix: process.argv.indexOf('--fix') > -1,
```

Comment:
I also thought about that and used `--fix-linter` previously. Let me change it to `--fix-lint` :)